.github/workflows/e2e-test.yml

```diff
@@ -0,0 +1,36 @@
+name: e2e
+
+on:
```

Comment:
It is better to allow manual execution for debugging.

Reply:
We should run on every pr and merge.

Reply:
@shaowenchen, Using a scheduled trigger won't block the PR/merge process when e2e failed. Because e2e depends on the ks-installer, It may fail caused by the ks-installer instead of the commit itself.